image-index.md

```diff
@@ -1,12 +1,12 @@
-# OCI Image Manifest List Specification
+# OCI Image Index Specification
 
 The manifest list is a higher-level manifest which points to specific [image manifests](manifest.md) for one or more platforms.
 While the use of a manifest list is OPTIONAL for image providers, image consumers SHOULD be prepared to process them.
```

Comment:
same here `While the use of a manifest list is OPTIONAL` 
and also https://github.com/opencontainers/image-spec/pull/546/files#diff-08e132d566e81340ee3533c41567d487R54 and https://github.com/opencontainers/image-spec/pull/546/files#diff-08e132d566e81340ee3533c41567d487R49